relay-server/src/actors/upstream.rs

```diff
@@ -835,6 +835,7 @@ impl SharedClient {
         &self,
         request: &mut dyn UpstreamRequest,
     ) -> Result<Response, UpstreamRequestError> {
+        request.configure(&self.config);
```

Comment:
Mh actually, I think we should only configure once or very very close to build and document when `configure` is called in the trait method docs.

Reply:
See this commit: 25a73d5fb7fecc2994bfe5fc7a718dc2ee48515e, this is exactly what's happening. I'll add a mention to the docs.